web/components/answers/create-answer-panel.tsx

```diff
@@ -163,3 +208,22 @@ export function CreateAnswerPanel(props: { contract: FreeResponseContract }) {
     </Col>
   )
 }
+
+type answerErrorLevel = 'warning' | 'error'
+
+const AnswerError = (props: { text: string; level: answerErrorLevel }) => {
+  const { text, level } = props
+  const colorClass =
```

Comment:
There's a code structure I like to make this kind of thing more concise & readable:
```
const colorClass= {
  error: 'text-red-500',
  warning: 'text-orange-500',
}[level] ?? ''
```

Reply:
Neat, I haven't seen that pattern before, but I like it